### PR TITLE
Plan details yearly monthly styles

### LINKED ISSUE #1856: Copy and style change for yearly/monthly payment
I've found this a little hard to understand as we have right now a toggle with just the work "monthly" or "yearly". Also when the toggle is toggled, there's no way to see if it's active or not. If it's active, it should have color IMHO. 

All in all, I think we should stick to the design
image

### PR DESCRIPTION
**Pull request by tanmoyAtb**
closes #1856

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

#### Theme
- [x] Components / elements inspected in light mode 
- [x] Components / elements inspected in dark mode

### CODE REVIEW

**Review by asnaith — APPROVED** [review]
This is working well 👍

I noticed this branch had conflicts that needed resolving and it was because I'd just merged some tests to dev (after your pr submission) that related to the plan details. I was checking to make sure the label changed when the toggle was enabled etc. 

I resolved the conflicts and updated the tests to reflect current behavior in this PR, thought it would be quicker for me to do it as I knew exactly what was going on.

**Comment by tanmoyAtb** [comment]
> This is working well 👍
> 
> I noticed this branch had conflicts that needed resolving and it was because I'd just merged some tests to dev (after your pr submission) that related to the plan details. I was checking to make sure the label changed when the toggle was enabled etc.
> 
> I resolved the conflicts and updated the tests to reflect current behavior in this PR, thought it would be quicker for me to do it as I knew exactly what was going on.

Thanks a lot for taking care of this ! The changes look good 👍

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PlanDetails.tsx [hunk 1]
```diff
@@ -191,7 +188,7 @@ const PlanDetails = ({ plan, goToSelectPlan, onSelectPlanPrice }: IPlanDetails)
             {billingPeriod === "monthly"
               ? `${monthlyPrice?.unit_amount ? monthlyPrice?.currency : ""} ${monthlyPrice?.unit_amount}`
               : `${yearlyPrice?.unit_amount ? yearlyPrice?.currency : ""} ${yearlyPrice?.unit_amount}`
-            }<span className={classes.normalWeightText}>{billingPeriod ? t`/month` : t`/year`}</span>
+            }<span className={classes.normalWeightText}>{billingPeriod === "monthly" ? t`/month` : t`/year`}</span>
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PlanDetails.tsx]: nit
```suggestion
            }
            <span className={classes.normalWeightText}>{billingPeriod === "monthly" ? t`/month` : t`/year`}</span>
```

**Review by Tbaut — COMMENTED** [review]
Looks good, just could we add the % discount next to the yearly billing as seen in the mock https://github.com/ChainSafe/ui-monorepo/issues/1856#issue-1102061579? That's very informative IMHO.

**Comment by tanmoyAtb** [comment]
> #1856 (comment)

Added the "% off" text, I have added a task to #1873 to make sure that the correct "% off" is displayed before deployment.

**Comment by Tbaut** [comment]
Thanks, what I meant is that we should calculate programmatically what percentage it is, we have both prices so that shouldn't be a problem to calculate,`((monthly * 12) - yearly)/(monthly * 12) * 100` so that we are consistent with what we show. Even if it shows 99% right now because our yearly plan is 1$, that's fine. I think this shouldn't be hard-coded :)

**Comment by tanmoyAtb** [comment]
> Thanks, what I meant is that we should calculate programmatically what percentage it is, we have both prices so that shouldn't be a problem to calculate,`((monthly * 12) - yearly)/(monthly * 12) * 100` so that we are consistent with what we show. Even if it shows 99% right now because our yearly plan is 1$, that's fine. I think this shouldn't be hard-coded :)

The percentage update is up !

**Review by Tbaut — APPROVED** [review]
looks great, thanks a lot!

**Review by FSM1 — APPROVED** [review]

**Pull request merged**